### ISSUE TITLE
Refactor: circular progress bar currently used in the app to working good with Next.js + Tailwind

### ISSUE THREAD

**Issue by heronlancellot** [w3b3d3v/web3task-front#144]
- We currently use the suspense component in Router, we should refactor the <SuspenseLoader /> to a new and better way, improved by **Next Js**. The current component uses a library called:
    - NProgress 
    - CircularProgress.

Proposal:

- [ ] Refactor the code to a **new** and _better way_, to loading when isn't ready yet.
- [ ] Refactor the library to use new implementations who doesn't need external libraries ([Check more about Next.js](https://nextjs.org/docs/app/building-your-application/routing/loading-ui-and-streaming))

```
const Loader = (Component) => (props) =>
(
  <Suspense fallback={<SuspenseLoader />}>
    <Component {...props} />
  </Suspense>
);
```
Or suggest a better one way !

### _Attention_ 
This task should be pushed to the **Next.js** _refactoring_ **branch** #16

**Comment by diegosano** [comment]
@heronlancellot hey, can I work on this one?

**Comment by heronlancellot** [comment]
hey @diegosano ! yeah, assigned to you bro

**Comment by heronlancellot** [comment]
closed by #159
